hasher-matcher-actioner/hmalib/lambdas/actions/writebacker.py

```diff
@@ -44,4 +45,16 @@ def lambda_handler(event, context):
 
 
 if __name__ == "__main__":
-    pass
+    # For basic debugging
+    # This will react to real descriptors if WRITEBACK_LOCAL is on
```

Comment:
Thanks for adding this! if you want to make it even more idiot proof for someone like me trying to use it later you could also add (probably want to test to make sure)
`# example usage: $ WRITEBACK_LOCAL=True python3 -m hmalib/lambdas/actions/writebaker.py`